proxy/http/HttpTransactHeaders.cc

```diff
@@ -1073,3 +1073,63 @@ HttpTransactHeaders::remove_privacy_headers_from_request(HttpConfigParams *http_
     }
   }
 }
+
+void
+HttpTransactHeaders::normalize_accept_encoding(const OverridableHttpConfigParams *ohcp, HTTPHdr *header)
+{
+  int normalize_ae = ohcp->normalize_ae;
+
+  if (normalize_ae == 99) { // Not explicitly configured.
+
+    if (ohcp->normalize_ae_gzip == 99) { // Not explicitly configured.
```

Comment:
Is this ever true? The default value is copied over to `ohcp` via `INT_TO_BOOL` which should turn `99` into `1`.